test/suites/integration/service_export_and_import_test.go

```diff
@@ -0,0 +1,93 @@
+package integration
+
+import (
+	"fmt"
+	"github.com/aws/aws-application-networking-k8s/pkg/latticestore"
+	"github.com/aws/aws-application-networking-k8s/test/pkg/test"
+	"github.com/aws/aws-sdk-go/service/vpclattice"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"log"
+	"os"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+var _ = Describe("Test service export and Service import", func() {
+	It("Create a k8s service, do service export and service import for it, refer this serviceImport in the httproute", func() {
+		gateway := testFramework.NewHttpGateway()
+		deployment, service := testFramework.NewHttpApp(test.HTTPAppOptions{Name: "service-import-export-test"})
+		serviceExport, serviceImport := testFramework.GetServiceExportAndServiceImportByService(service)
+
+		httpRoute := testFramework.NewPathMatchHttpRoute(gateway, []client.Object{service, serviceImport}, "http")
+		testFramework.ExpectCreated(ctx,
```

Comment:
pathmatch0 refer to a Service and /pathmatch1 refer to a ServiceImport. Both Service and ServiceImport refer the same targetGroup